index.html

```diff
@@ -323,5 +1235,272 @@ <h3>Reference</h3>
+      transition: color 0.3s ease;
+    }
+
+    .nav-link::after {
+      content: '';
+      position: absolute;
+      bottom: 0;
+      left: 50%;
+      width: 0;
+      height: 2px;
+      background: #26a69a;
+      transition: all 0.3s ease;
+      transform: translateX(-50%);
+    }
+
+    .nav-link:hover::after {
+      width: 100%;
+    }
+
+    /* List Item Animations */
+    .mission-points li, .benefits-list li {
+      transition: all 0.3s ease;
+      position: relative;
+      padding-left: 25px;
+    }
+
+    .mission-points li:hover, .benefits-list li:hover {
+      transform: translateX(10px);
+      color: #26a69a;
+    }
+
+    .mission-points li::before, .benefits-list li::before {
+      content: '•';
+      position: absolute;
+      left: 0;
+      color: #26a69a;
+      transition: transform 0.3s ease;
+    }
+
+    .mission-points li:hover::before, .benefits-list li:hover::before {
+      transform: scale(1.5);
+    }
+
+    /* Divider Animations */
+    .divider {
+      position: relative;
+      overflow: hidden;
+    }
+
+    .divider::after {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: -100%;
+      width: 100%;
+      height: 100%;
+      background: linear-gradient(90deg, transparent, rgba(38, 166, 154, 0.2), transparent);
+      animation: dividerShine 3s infinite;
+    }
+
+    /* Image Animations */
+    img:not(.logo-img) {
+      transition: transform 0.3s ease;
+    }
+
+    img:not(.logo-img):hover {
+      transform: scale(1.05);
+    }
+
+    /* Scroll Animations */
+    @keyframes fadeIn {
+      from { opacity: 0; transform: translateY(20px); }
+      to { opacity: 1; transform: translateY(0); }
+    }
+
+    .scroll-fade {
+      opacity: 0;
+      transform: translateY(20px);
+      transition: all 0.6s ease;
+    }
+
+    .scroll-fade.visible {
+      opacity: 1;
+      transform: translateY(0);
+    }
+
+    /* Add scroll animation functionality */
+    <script>
+      document.addEventListener('DOMContentLoaded', function() {
+        const scrollElements = document.querySelectorAll('.scroll-fade');
+        
+        const elementInView = (el, percentageScroll = 100) => {
+          const elementTop = el.getBoundingClientRect().top;
+          return (
+            elementTop <= 
+            ((window.innerHeight || document.documentElement.clientHeight) * (percentageScroll/100))
+          );
+        };
+        
+        const displayScrollElement = (element) => {
+          element.classList.add('visible');
+        };
+        
+        const handleScrollAnimation = () => {
+          scrollElements.forEach((el) => {
+            if (elementInView(el, 90)) {
+              displayScrollElement(el);
+            }
+          });
+        };
+        
+        window.addEventListener('scroll', () => {
+          handleScrollAnimation();
+        });
+        
+        // Initial check
+        handleScrollAnimation();
+      });
+    </script>
+  </style>
```

Comment:
```suggestion
```

As far as I know, the HTML <script> element is not designed to be placed inside (!) the <style> element?